tests/core/test_hdaccount.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+
+from eth_utils import (
+    ValidationError,
+)
+
+from eth_account import (
+    Account,
+)
+from eth_account.hdaccount import (
+    ETHEREUM_DEFAULT_PATH,
+)
+
+Account.enable_unaudited_hdwallet_features()
+
+
+@pytest.mark.parametrize("mnemonic,account_path,expected_address", [
+    # Ganache
+    # https://github.com/trufflesuite/ganache-core/blob/master/test/accounts.js
```

Comment:
permalink please

https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files